.travis.yml

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright 2017 HyperHQ Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+sudo: required
+dist: xenial
```

Comment:
why not bionic ?

Reply:
I copied from another repo - I'll go with whatever - so, let me switch it to bionic... np

Reply:
for ref then, I think there are still a few repos using xenial in their Travis. iirc, I copied from the agent (I wanted a 'simple' template).
https://github.com/search?q=org%3Akata-containers+xenial&type=Code

Reply:
yeah, I think all other repos still use xenial, would be good to later make the update in those places.